src/List/ListItem.js

```diff
@@ -54,6 +59,14 @@ export const styleSheet = createStyleSheet('MuiListItem', (theme) => {
   };
 });
 
+const mapListIemChildren = (children, classes, dense) => React.Children.map(children, (child) => {
```

Comment:
@oliviertassinari Definitely **context** will rock here ☺️  my lack of experience is making me find hard ways first 😂